src/recorder.js

```diff
@@ -131,6 +130,7 @@ class Recorder {
   append (timestamp, record) {
     var self = this
     self.data.journal.push({ timestamp, record })
+    self.event.trigger('showtxRecordedCount', [self.data.journal.length])
```

Comment:
perhaps the name of the event should more general, like `newTxRecorded`, and parameters would be `[timestamp, record, self.data.journal.length]` ?

Reply:
Ok